Makefile

```diff
@@ -1,5 +1,5 @@
 # Image URL to use all building/pushing image targets
-IMG ?= gcr.io/k8s-staging-cluster-api-huawei/cluster-api-huawei-controller:latest
+IMG ?= ghcr.io/huaweiclouddeveloper/cluster-api-huawei-controller:latest
```

Comment:
Change tag to get tag later for real releases.

The current is building images for master-dev